monitoring/uss_qualifier/scd/executor/test_steps/inject_flight.py

```diff
@@ -18,6 +19,15 @@
 def execute(runner, step: TestStep, step_ref: TestStepReference, target: TestTarget) -> None:
     print("[SCD]     Step: Inject flight {} to {}".format(step.inject_flight.name, target.name))
     try:
+        reference_time = datetime.utcnow() + timedelta(seconds=pytimeparse.parse(step.inject_flight.planning_time))
```

Comment:
This is different adjustment logic than I would have expected.  My expectation is that we would offset all volume timestamps (time_start and time_end) by the same timedelta, and that timedelta would be now - reference_time + planning_time.  Adding that timedelta makes it so that a volume timestamp equal to reference_time in the automated test definition would become (now + planning time) after adjustment.